src/containers/GuestInfoContainer.tsx

```diff
@@ -58,7 +58,7 @@ export const GuestInfoContainer = () => {
 
   const methods = useForm<PersonalInfo>({
     resolver: yupResolver(NewUserSchema),
```

Comment:
Why we are using the yup package for object validation?
Our glider-types package contains JSON schema for the Guest type, so we can validate objects using the schema. OK, if you do not like a schema validator helper from the `@windingtree/org.id-utils` package you always can use the original the `ajv` package.
If the original Guest info definition (from the win types) will be changed you will be required to refactor your yup code but  if  you will use JSON schema based validation you should not touch anything
If usage of yup is mandatory for the useForm hook you have to use types from the schema at least in the yup resolver definition.

Reply:
Can we leave it to refactor pr?

Reply:
Do not change it for now.